components/Hero/Hero.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from './hero.module.scss'
+import  Wave  from '../Wave/Wave'
+
+const Hero = () => {
+    return (
+      <div className={styles.hero__content_blue}>
+         <div className="content__dark py-4">
+           <div className="container d-flex flex-column flex-md-row justify-content-between align-items-center">
+             <div className="order-md-1 ">
+               <div className={styles.hero}>
+                 <div className="hero__text pe-4 text-end w-100">
+                   <p>Hello I{`'`}m</p>
+                   <h1>Alba Lucía González</h1>
+                 </div>
+               </div>
+             </div>
+             <img src='AlbaPortafolio.png' alt='Photo Alba Lucia Gonzalez' className={styles.hero__image}/>
```

Comment:
hmmm you probably only need to insert the style once at the top... if you call the component by the correct name, then everything can come underneath the .hero class when you write the SCSS.

Look at the scss file example I am going to write for you :)